### ISSUE TITLE
Not Staking

### ISSUE THREAD

**Issue by Raito00** [bitconnectcoin/bitconnectx#3]
How many days need mature coins ?

**Comment by BlTC0lN** [comment]
you need to wait 15 days to mature the coins.   You also need to make sure the wallet is unlocked after the 15 days to earn rewards

**Comment by Raito00** [comment]
Already  17 Days - still no mature coins. Wallet unlocked and 64 active connections

**Comment by BlTC0lN** [comment]
have you left it unlocked and running for at least 8 consecutive hours after the 15 day period?

**Comment by Raito00** [comment]
YES wallet is open and unlocked for staking 24/7

**Comment by BlTC0lN** [comment]
On the bottom of your wallet, on the right, there are 4 icons.
The one on the far left looks like a lock, the one on the far right is a green check mark.
Roll your mouse over the 2nd from the left, that looks like an X.  What does it say?

Does it say "not staking becuase you don't have mature coins" ?
or does it say something else?

FYI  my coins are only 11 days old so I am not staking BCCX yet, but i am staking BCC

**Comment by Raito00** [comment]
Yes on corner it say  "not staking becuase you don't have mature coins"
So looks like BCCX in this wallet version dont have staking option .. :(

**Comment by Raito00** [comment]
Ok - this probably not a issue... because out there is no information about BCCX PoS configuration ... can be also need mature coins 30 days or more ... or this first wallet version is just receive and send coins...

**Comment by BlTC0lN** [comment]
Has anyone been able to stake yet?  I have not

**Comment by Widi92** [comment]
For now it looks like only the addresses with more than 10k bccx are able to stake. You can look it up at http://explorer.bitconnectxpool.com
